clouds/aws/aws_roles/rhoam_add_role_policy.py

```diff
@@ -0,0 +1,20 @@
+from aws import aws_clients
```

Comment:
The repo is for cloud tools; so rhoam-specific code should not be in this repo.
This file should be delete (and code used where needed); json file should be moved next to the new code. (also remove text from README)

Reply:
No problem- so which exact location did [this](https://github.com/RedHatQE/cloud-tools/pull/14#:~:text=for%20rhoam%20%2D%20please%20add%20ref%20to%20the%20docs) referred to?
_for rhoam - please add ref to the docs_